lib/pages/main/home_page.dart

```diff
@@ -36,7 +32,7 @@ class _HomePageState extends State<HomePage> {
   void initState() {
     super.initState();
     initAuthStatus();
-    //_getNotificationCount();
+    _getNotificationCount();
```

Comment:
I guess this method is already in main_vc.dart from pull https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/141